src/support/auth/handlers/ims.js

```diff
@@ -67,7 +73,10 @@ export default class AdobeImsHandler extends AbstractHandler {
 
   async #getJwksUri(config) {
     if (!this.jwksCache) {
-      this.jwksCache = createRemoteJWKSet(new URL(config.discovery.jwks_uri));
+      /* c8 ignore next 3 */
```

Comment:
remote key set not testable currently as nock no longer is able to intercept `jose` http requests with more recent node versions.